screens/GameDetails.js

```diff
@@ -0,0 +1,301 @@
+  
+        const organizerRes = await getGameOrganizer(playerRes);
+        setGameOrganizer(organizerRes);
+  
+        const invitedPlayersRes = await getInvitedPlayers();
+        setAccepted(invitedPlayersRes.accepted);
+        setDeclined(invitedPlayersRes.declined);
+  
+        setLoading(false);
+      } catch (error) {
+        console.log("Error fetching data: ", error);
+        setLoading(false);
+      }
+    }
+  
+    fetchData();
+  }, []);
+
+
+  function isGameOwner() {
+    // Use a dummy username until actual organizer names are used
+    // We would need DUMMY_USERNAME to be the name of the current user
+    return thisPlayer.id == details.organizer;
+  }
+
+  function showDescription() {
+    return details.description != "" && details.description != undefined;
+  }
+
+  const renderItem = ({ item, index }) => {
+    return (
+      <View key={index} style={styles.nameContainer}>
+        <Text style={styles.text}>{item.name}</Text>
+        <Text
+          style={[
+            styles.text,
+            item.status == "In" ? { color: "green" } : { color: "red" },
+          ]}
+        >
+          {item.status}
+        </Text>
+      </View>
+    );
+  };
+
+  function handleEdit() {
+    navigation.navigate(("UpdateGame"), {
+        game: route.params.item,
+      });
+  }
+ 
+  async function handleDelete() {
+    setLoading(true);
+    const toDelete = await DataStore.query(Game, details.id);
+    DataStore.delete(toDelete);
+    setLoading(false);
+    navigation.navigate("HomeScreen");
+  }
+
+  return (
+    <View style={styles.container}>
+      <BackArrow location="HomeScreen" />
+      <View style={styles.infoContainer}>
+        <Text style={styles.topText}>{details.name}</Text>
+        <Text style={styles.text}>
+          <Text style={styles.bold}>Date: </Text>
+          {epochToLocalDate(details.datetime)}
+        </Text>
+        <Text style={styles.text}>
+          <Text style={styles.bold}>Location: </Text>
+          {details.location}
+        </Text>
+        <Text style={styles.text}>
+          <Text style={styles.bold}>Time: </Text>
+          {epochToLocalTime(details.datetime)}
+        </Text>
+        <Text style={styles.text}>
+          <Text style={styles.bold}>Organizer: </Text>
+          {loading ? "Loading..." : gameOrganizer}
+        </Text>
+        {showDescription() && (
+          <Text style={styles.text}>
+            <Text style={styles.bold}>Description: </Text>
+            {details.description}
+          </Text>
+        )}
+      </View>
+      <View style={[styles.infoContainer, styles.acceptedContainer]}>
+        <FlatList
+          ListHeaderComponent={
+            <>
+              <Text style={[styles.text, styles.bold]}>Players Attending</Text>
+            </>
+          }
+          data={[...accepted, ...declined]}
+          renderItem={renderItem}
+        />
+      </View>
+      {!isGameOwner() ? <View style={styles.row}>
```

Comment:
Since this is now conditionally rendering either the button container or the RSVP bar, make sure the RSVP bar container has absolute positioning at the bottom of the screen exactly like the button container does. This will ensure that the spacing of the button container vs the RSVP bar is consistent.